hoot-core-test/src/test/cpp/hoot/core/ops/FindIntersectionsOpTest.cpp

```diff
@@ -62,6 +62,11 @@ class FindIntersectionsOpTest : public CppUnit::TestFixture
 
 public:
 
+  void setUp()
```

Comment:
Are these core test changes part of making the setup easier?  Or did they come from some other branch?  Either way is fine, I just don't know enough about the core tests to give them a good review.